src/utils.jl

```diff
@@ -42,10 +42,19 @@ Save the fitted model in `estimator` as `filename`.
 * `filename::String`: the name of the file to save the model in.
 * `num_iteration::Integer`: keyword argument that sets the number of iterations of the model that
     should be saved. `< 0` for all iterations.
+* `start_iteration` : : Start index of the iteration that should be saved.
+* `feature_importance_type` : Type of feature importance,
+    can be C_API_FEATURE_IMPORTANCE_SPLIT or C_API_FEATURE_IMPORTANCE_GAIN
 """
```

Comment:
You probably want to add "average_precision" to the `MAXIMIZE_METRICS` constant above (and probably review it for any other maximisable metrics)

Reply:
f045a87